docs/index.rst

```diff
@@ -105,7 +106,8 @@ The API supporting ``docker search`` can be enabled by configuring a Solr
 deployment for use by crane. In crane's configuration file, a section
 ``[solr]`` must exist with key ``url``. The URL will be used in a GET request,
 and it must contain the string ``{0}`` as a placeholder where the search string
-will be inserted.
+will be inserted. The search returns both results containing both V1
```

Comment:
can you please add a note that search for v2 works just with Solr.

Reply:
Solr works with V1 and V2. GSA works with V1 only.